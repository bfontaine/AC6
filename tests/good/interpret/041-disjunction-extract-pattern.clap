val classify = case { A[x] or B[x] => x | C => 0 }
val xA = classify (A[42])
val xB = classify (B[42])
val xC = classify (C)

