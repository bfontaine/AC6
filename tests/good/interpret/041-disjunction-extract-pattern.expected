:- classify = <code>
:- xA = 42
:- xB = 42
:- xC = 0
